src/script/lua_api/l_vmanip.cpp

```diff
@@ -34,13 +34,19 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 
 extern "C" int VoxelManip_get_volume(void **lvmp)
 {
+	if (lvmp == nullptr)
```

Comment:
Shouldn't you check that `*lvmp` is non-null as well?

Reply:
It's easy to pass a nil value by mistake, but I can't imagine how to pass a userdata container with a pointer pointing to null to this function via Lua. I didn't feel it necessary to guard against that. I can do it if you think it is.

Reply:
No, you're right -- just leave it as it is.